### DISCUSSION TITLE
Mapping OK?

### DISCUSSION THREAD

**Discussion by Pewibe** [gemu2015/Sonoff-Tasmota#50] [Q&A] [answered]
This test script for mapping:
```
>D 
x=0
y=0

>B
y=mp(x <50:25 <100:50 >100:0)
print _%x%_%y%_
```
produces the console output:
02:05:26.465 Script: nv=2, tv=0, vns=4, vmem=46, smem=8192
02:05:26.468 var not found: :25 <100:50 >100:0)
02:05:26.470 _0.00_0.00_

1. Do I misunderstand the doc and script is wrong?
2. What about a comparison oper like >=? Do I have to use 2 separate comparisons?
3. In doc, by "numeric value v1, v2 a number is mandatory? Using a variable having a numeric value assigned ist not possible?

**Comment by gemu2015** [comment, marked as answer]
this function including documentation was a contribution from an external coder. i never used it.
1. the colon is wrong, use space instead
2. no support for >=
3. you may assign variables to numeric parameters
